wrangler/src/trust/trust.js

```diff
@@ -0,0 +1,271 @@
+import { Keypair, TransactionBuilder, Networks, BASE_FEE, Operation, Account } from 'stellar-base'
+import { parse } from 'toml'
+import localtoml from '../turret/toml'
+
+/**
+ * It gets the account info for a given public key.
+ * @param pubkey - The public key of the account you want to get information about.
+ * @returns The JSON response from the Horizon API.
+ */
+export async function getAccountInfo(pubkey){
+    const accountInfo = (await fetch(`${HORIZON_URL}/accounts/${pubkey}`))
+    return accountInfo.json()
+}
+
+/**
+ * It gets all the accounts that are signed on by the given public key.
+ * @param pubkey - The public key of the signer you want to get the accounts of.
+ * @returns An array of account IDs that are signed by the given public key.
+ */
+export async function getAccountsOfSigner(pubkey) {
+    const signedOnAccounts = []
+    const getrecords = async(url) => await fetch(url)
+    const signerinfo = await (await getrecords(`${HORIZON_URL}/accounts/?signer=${pubkey}&limit=200`)).json()
+    signerinfo._embedded.records.map((record) => {
+        signedOnAccounts.push(record.account_id)
+    })
+    return signedOnAccounts
+}
```

Comment:
unused

Reply:
correct this is unused. there may be one other unused util function also, they were used in my create source account call which was decided to not be included, thus any unused calls can be removed.